tools/bin/hawqregister

```diff
@@ -855,6 +895,85 @@ class HawqRegister(object):
         logger.info('Hawq Register Succeed.')
 
 
+class HawqRegisterPartition(HawqRegister):
+    def _get_partition_info(self):
+        dic = {}
+        for ele in self.accessor.get_partition_info(self.dst_table_name):
+            dic[ele['partitionboundary']] = ele['partitiontablename']
+        return dic
+
+    def _mapping_tablename_from_yml(self, partitions):
+        ''' Mapping table name from yml file, return a list of (table_name,(file_path, file_size)) '''
+        mappings = []
+        for pos, constraint in enumerate(self.partitions_constraint):
+            if partitions.has_key(constraint):
+                mappings.extend([(partitions[constraint], (self.partitions_filepaths[pos][i], self.partitions_filesizes[pos][i]))
```

Comment:
Do we have processing mechanism for un-expected error? For example, filepaths[pos][i] has value which filesizes[pos][i] value is invalid?